src/ClientModel/SamplingDataPostProcessorTest.cpp

```diff
@@ -122,18 +122,17 @@ MATCHER_P(SortedCallstackReportEq, that, "") {
   // descending order), hence the order is not unique. Sort again considering
   // `CallstackCount::callstack_id`, too, to facilitate the comparison of `SortedCallstackReport`s
   // for equality.
-  auto callstack_count_less = [](const CallstackCount& lhs, const CallstackCount& rhs) {
-    if (lhs.count != rhs.count) return lhs.count > rhs.count;
-    return lhs.callstack_id < rhs.callstack_id;
+  auto callstack_count_projector = [](const CallstackCount& callstack_count) {
+    return std::make_pair(callstack_count.count, callstack_count.callstack_id);
   };
 
   std::vector<CallstackCount> lhs_resorted_callstacks_counts = lhs.callstack_counts;
-  std::sort(lhs_resorted_callstacks_counts.begin(), lhs_resorted_callstacks_counts.end(),
-            callstack_count_less);
+  orbit_base::sort(lhs_resorted_callstacks_counts.begin(), lhs_resorted_callstacks_counts.end(),
+                   callstack_count_projector, std::greater<>{});
```

Comment:
Does using `std::greater<>{}` make a difference here? It's already not sorting like the original comparator anyway because that one was sorting by decreasing count but increasing id.